src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -699,6 +699,190 @@ class ContentSidebar extends PureComponent<Props, State> {
+     * @param {Function} errorCallback - the function which will be called on error
+     * @param {string} dueAt - The date the task is due
+     * @return {void}
+     */
+    updateTask = (
+        taskId: string,
+        message: string,
+        successCallback: (task: Task) => void = noop,
+        errorCallback: (e: Error) => void = noop,
+        dueAt?: string
+    ) => {
+        const { file } = this.state;
+        const { onTaskUpdate = noop } = this.props;
+
+        if (!file) {
+            throw getBadItemError();
+        }
+
+        this.api.getTasksAPI(false).updateTask({
+            file,
+            taskId,
+            message,
+            dueAt,
+            successCallback: (task: Task) => {
+                onTaskUpdate(task);
+                successCallback(task);
+                this.updateTaskSuccessCallback(task);
+            },
+            errorCallback: (e: Error) => {
+                errorCallback(e);
+                this.errorCallback(e);
+            }
+        });
+    };
+
+    /**
+     * Task update success callback
+     *
+     * @private
+     * @param {Object} task - Box task
+     * @return {void}
+     */
+    updateTaskSuccessCallback(task: Task) {
+        const { tasks } = this.state;
+
+        if (tasks) {
+            const { entries, total_count: totalCount } = tasks;
+
+            this.setState({
+                tasks: {
+                    entries: [task, ...entries],
+                    total_count: totalCount + 1
+                }
+            });
+        }
+    }
+
+    /**
+     * Deletes a task
+     *
+     * @private
+     * @param {string} taskId - The task's id
+     * @param {Function} successCallback - the function which will be called on success
+     * @param {Function} errorCallback - the function which will be called on error
+     * @return {void}
+     */
+    deleteTask = (
+        taskId: string,
+        successCallback: (taskId: string) => void = noop,
+        errorCallback: (e: Error, taskId: string) => void = noop
+    ) => {
+        const { file } = this.state;
+        const { onTaskDelete = noop } = this.props;
+
+        if (!file) {
+            throw getBadItemError();
+        }
+
+        this.api.getTasksAPI(false).deleteTask({
+            file,
+            taskId,
+            successCallback: () => {
+                onTaskDelete(taskId);
+                successCallback(taskId);
+                this.deleteTaskSuccessCallback(taskId);
+            },
+            errorCallback: (e: Error) => {
+                errorCallback(e, taskId);
+                this.errorCallback(e);
+            }
+        });
+    };
+
+    /**
+     * Task update success callback
+     *
+     * @private
+     * @param {Object} task - Box task
+     * @return {void}
+     */
+    deleteTaskSuccessCallback = (taskId: string) => {
+        const { tasks } = this.state;
+
+        if (tasks) {
+            const { entries } = tasks;
+
```

Comment:
You love them blank lines, don't you. Feel a bit random, to have it between some const declarations and not others.

Reply:
Yeah I guess typically I put blank lines in between variable declarations and if statements/function calls. Other pieces of the code have random spaces as well. Is there something that can be done in eslint / prettier to make this consistent?